openwisp_controller/config/templates/admin/config/system_context.html

```diff
@@ -1,8 +1,17 @@
-{% for key, value in system_context.items %}
-    {% if new_line in value %}
-        <p><strong>{{ key }}:</strong><pre>{{ value }}</pre></p>
-    {% else %}
-        <p><strong>{{ key }}:</strong> {{ value }}</p>
-    {% endif %}
-{% endfor %}
+<div> 
+    <div class="system-context">
+        {% if  system_context|length > 0%}
+        {% for key, value in system_context.items %}
+            {% if new_line in value %}
+                <p><strong>{{ key }}:</strong><pre>{{ value }}</pre></p>
```

Comment:
`<pre>` tags cannot be contained in `<p>`elements according to the HTML spec, please use `<div>` instead of `<p>` in both occurrences (the second one for consistency).